nginx_proxy/start_nginx.sh

```diff
@@ -31,30 +32,47 @@ configuration is already provided on disk:
 $(command basename $0)
 (2) Starts nginx with a custom Endpoints service name and service version, and
 have nginx obtain the service configuration:
-$(command basename $0) -n ENDPOINTS_SERVICE_NAME -v ENDPOINTS_SERVICE_VERSION
+$(command basename $0) -n ENDPOINTS_SERVICE_NAME \
+-v ENDPOINTS_SERVICE_VERSION -r ROLLOUT_STRATEGY
```

Comment:
Since this is an example of providing a service version, then it should be "-r fixed".

An example (3) should be added to show how to run it in managed mode.